### ISSUE TITLE
error in 'Version' field string 'v0.0.3': version number does not start with digit

### ISSUE THREAD

**Issue by iinstances** [radicle-dev/radicle#627]
```
dpkg -i radicle_latest_amd64.deb 
dpkg: error processing archive radicle_latest_amd64.deb (--install):
 parsing file '/var/lib/dpkg/tmp.ci/control' near line 2 package 'radicle':
 **error in 'Version' field string 'v0.0.3': version number does not start with digit**
Errors were encountered while processing:
 radicle_latest_amd64.deb
```

**Comment by iinstances** [comment]
**--force-bad-version** do the work!
`dpkg --force-bad-version -i radicle_latest_amd64.deb`

**Comment by jkarni** [comment]
This should be fixed with #628 - can you try installing the latest version (0.0.4)?

**Comment by jkarni** [comment]
@jameshaydon just verified the fix. Closing.
